osu.Game/Overlays/Comments/VotePill.cs

```diff
@@ -62,16 +64,20 @@ private void load(OsuColour colours)
             AccentColour = borderContainer.BorderColour = sideNumber.Colour = colours.GreenLight;
             hoverLayer.Colour = Color4.Black.Opacity(0.5f);
 
-            if (api.IsLoggedIn && api.LocalUser.Value.Id != comment.UserId)
+            var ownComment = api.LocalUser.Value.Id == comment.UserId;
+
+            if (api.IsLoggedIn && !ownComment)
                 Action = onAction;
+
+            Background.Alpha = ownComment ? 0 : 1;
```

Comment:
I think it's weird that this is still in a visually clickable state when not logged in? In `osu-web`, this triggers the login pane to display, so hopefully we'd do the same here.

Reply:
Fixed in https://github.com/ppy/osu/pull/11572/commits/20161aea6a472320f34ff0e8057d08a0cdea527d